src/imitation/data/rollout.py

```diff
@@ -358,10 +358,16 @@ def rollout_stats(trajectories: Sequence[types.TrajectoryWithRew]) -> Dict[str,
         "len": np.asarray([len(t.rews) for t in trajectories]),
     }
 
-    infos_peek = trajectories[0].infos
-    if infos_peek is not None and "episode" in infos_peek[-1]:
-        monitor_ep_returns = [t.infos[-1]["episode"]["r"] for t in trajectories]
+    monitor_ep_returns = []
+    for t in trajectories:
+        if t.infos is not None:
+            ep_return = t.infos[-1].get("episode", {}).get("r")
+            if ep_return:
```

Comment:
If I understand the code correctly, this should be `if ep_return is not None`? Otherwise, we would skip episodes that happen to have exactly zero return